### ISSUE TITLE
Error: `No entity id specified for entity 100 Registered`

### ISSUE THREAD

**Issue by felipecrs** [TECH7Fox/asterisk-hass-integration#70]
This showed up in my HA logs:

![image](https://user-images.githubusercontent.com/29582865/208919152-59d392fb-d08b-4be0-bfc0-75850ccdb0ae.png)

**Comment by TECH7Fox** [comment]
Weird. Is it constant or random?

**Comment by felipecrs** [comment]
Happens almost every time I restart HA. But sometimes it's not 100, I already saw it being 200.

**Comment by TECH7Fox** [comment]
Is this still a issue in v1.0.0?

**Comment by felipecrs** [comment]
Apparently not, but there is a new one:

```
Logger: custom_components.asterisk
Source: custom_components/asterisk/__init__.py:43
Integration: Asterisk (documentation, issues)
First occurred: 17:28:03 (1 occurrences)
Last logged: 17:28:03

Devices complete: Event : EndpointListComplete -> {'ActionID': '2', 'EventList': 'Complete', 'ListItems': '7'}
```

**Comment by TECH7Fox** [comment]
Ah, forgot to remove that. Fixed in v1.0.2.

**Comment by TECH7Fox** [comment]
Will close now. Let me know if you get any issues.
